src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportService.cs

```diff
@@ -99,8 +99,31 @@ public async Task RemoveImportTableRowIfExists(Guid releaseId, string dataFileNa
 
         public async Task FailImport(Guid releaseId, string dataFileName, IEnumerable<ValidationError> errors)
         {
+            var release = _context.Releases
+                .Where(r => r.Id.Equals(releaseId))
+                .Include(r => r.Publication)
+                .ThenInclude(p => p.Topic)
+                .ThenInclude(t => t.Theme)
+                .FirstOrDefault();
+
+            var importReleaseMessage = JsonConvert.SerializeObject(new ImportMessage()
+            {
+                SubjectId = Guid.Empty,
+                DataFileName = dataFileName,
+                OrigDataFileName = "Unknown",
+                MetaFileName = "Unknown",
+                Release = _mapper.Map<Release>(release),
+                NumBatches = 0,
+                BatchNo = 0,
+                RowsPerBatch = 0,
+                Seeding = false,
+                TotalRows = 0,
+                ArchiveFileName = "Unknown"
```

Comment:
Would be better to leave it as an empty string or null. 'Unknown' could literally be the name of a file :wink: